.github/workflows/PR.yml

```diff
@@ -0,0 +1,26 @@
+name: Label and Assign PRs
+on:
+  pull_request:
+    types:
+      - opened
+jobs:
+  label_and_assign_PRs:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - run: |
+          gh pr edit "$NUMBER" --add-label "$LABELS"
+          gh pr edit "$NUMBER" --add-assignee "$ASSIGNEE"
+          curl \
```

Comment:
Add this line to `curl` to capture the HTTP response code:
`-w %{http_code} \`

Can ignore the `\` if at the end of `curl` call.